docs/additional-functionality/advanced_configs.md

```diff
@@ -210,12 +211,13 @@ Name | SQL Function(s) | Description | Default Value | Notes
 <a name="sql.expression.BitwiseNot"></a>spark.rapids.sql.expression.BitwiseNot|`~`|Returns the bitwise NOT of the operands|true|None|
 <a name="sql.expression.BitwiseOr"></a>spark.rapids.sql.expression.BitwiseOr|`\|`|Returns the bitwise OR of the operands|true|None|
 <a name="sql.expression.BitwiseXor"></a>spark.rapids.sql.expression.BitwiseXor|`^`|Returns the bitwise XOR of the operands|true|None|
+<a name="sql.expression.BloomFilterMightContain"></a>spark.rapids.sql.expression.BloomFilterMightContain| |Bloom filter query|true|None|
 <a name="sql.expression.BoundReference"></a>spark.rapids.sql.expression.BoundReference| |Reference to a bound variable|true|None|
 <a name="sql.expression.CaseWhen"></a>spark.rapids.sql.expression.CaseWhen|`when`|CASE WHEN expression|true|None|
 <a name="sql.expression.Cast"></a>spark.rapids.sql.expression.Cast|`bigint`, `binary`, `boolean`, `cast`, `date`, `decimal`, `double`, `float`, `int`, `smallint`, `string`, `timestamp`, `tinyint`|Convert a column of one type of data into another type|true|None|
 <a name="sql.expression.Cbrt"></a>spark.rapids.sql.expression.Cbrt|`cbrt`|Cube root|true|None|
-<a name="sql.expression.Ceil"></a>spark.rapids.sql.expression.Ceil|`ceil`, `ceiling`|Ceiling of a number|true|None|
-<a name="sql.expression.CheckOverflow"></a>spark.rapids.sql.expression.CheckOverflow| |CheckOverflow after arithmetic operations between DecimalType data|true|None|
+<a name="sql.expression.Ceil"></a>spark.rapids.sql.expression.Ceil| |Ceiling of a number|true|None|
+<a name="sql.expression.CheckOverflowInTableInsert"></a>spark.rapids.sql.expression.CheckOverflowInTableInsert| |Casting a numeric value as another numeric type in store assignment|true|None|
```

Comment:
this seems a little odd. .Do we need to re-gen?

Reply:
I tried to regen just now and it still added this. Could it be that it was missed in a previous change?

Reply:
It looks like this docstring comes from some shims files. Maybe I am supposed to build the code against 320 for doc gen purposes or something? Not sure